core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -330,17 +331,28 @@ var ExplorationEditorMainTab = function() {
        * Check for correct rule parameters.
        * @param {string} [interactionId] - Interaction type.
        * @param {string} [ruleName] - Appropriate rule of provided interaction.
+       * @param {integer} [ruleIndex] - The index of the rule to check.
        * @param {string[]} [feedbackTextArray] - Exact feedback text to match.
        */
-      expectRuleToBe: function(interactionId, ruleName, feedbackTextArray) {
-        var ruleDescription = _getRuleDescription(interactionId, ruleName);
-        // Replace selectors with feedbackTextArray's elements.
-        ruleDescription = _replaceRuleInputPlaceholders(
-          ruleDescription, feedbackTextArray);
-        ruleDescription += '...';
-        // Adding "..." to end of string.
-        var answerTab = element(by.css('.protractor-test-answer-tab'));
-        expect(answerTab.getText()).toEqual(ruleDescription);
+      expectRuleToBe: function(
+          interactionId, ruleName, ruleIndex, feedbackTextArray) {
+        getAnswerTabs.then(
```

Comment:
Typically, if you are calling this getAnswerTabs then there should be a function call at the end of it: getAnswerTabs() (since it should be a method).

Reply:
done